hack/make/build-deb

```diff
@@ -86,6 +86,16 @@ set -e
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
 			RUN cp -aL hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
+		EOF
+		# Remove the following case-based substitution when none of these are supported, and the TasksMax value is uncommented in docker.service
+		case "$version" in
+		debian-jessie|debian-wheezy|ubuntu-precise|ubuntu-trusty|ubuntu-wily)
+			;;
+		*)
+			echo "RUN sed -i -- 's/#TasksMax=infinity/TasksMax=infinity/' contrib/init/systemd/docker.service" >> "$DEST/$version/Dockerfile.build"
```

Comment:
Sorry to disturb once more. I've done a `make deb` with the 1.12.0-rc2 tag and checked both debian-jessie and debian-stretch for amd64.

The docker binary for debian-stretch shows the "unsupported" string and the debian-jessie binary is fine:

```
root@00b88863593e:/tmp/jessie/usr/bin# ./docker --version
Docker version 1.12.0-rc2, build 906eacd

root@00b88863593e:/tmp/stretch/usr/bin# ./docker --version
Docker version 1.12.0-rc2, build 906eacd-unsupported
```

So how should we proceed with fix for raspbian? I would like to send a PR to revert this one as it does not seem to work as intended as the `sed` changes the source tree and the binaries in the deb package are marked as unsupported.